app/models/miq_expression/relative_datetime.rb

```diff
@@ -10,6 +10,11 @@ def self.relative?(value)
     v.starts_with?("this", "last") || v.ends_with?("ago") || ["today", "yesterday", "now"].include?(v)
   end
 
+  def self.normalize_date(is_date, rel_time, tz, mode = "beginning")
```

Comment:
Bearing in mind there is already a `RelativeDatetime.normalize` method, does `.normalize_date` fully describe its use? I'm also noticing that it receives an argument called `rel_time` and returns either a date or a time